process/block/shardblock.go

```diff
@@ -1569,6 +1574,11 @@ func (sp *shardProcessor) createMiniBlocks(
 		log.Debug(errNotCritical.Error())
 	}
 
+	log.Info(fmt.Sprintf("processed metaBlock %d out of %d", len(processedMetaHdrs), len(usedMetaHdrsHashes)))
```

Comment:
Remove these prints which were uses only for quick debugging purpose

Reply:
done